packages/netlify/src/index.ts

```diff
@@ -242,7 +254,7 @@ export default function netlifyIntegration(
 					},
 					image: {
 						service: {
-							entrypoint: isRunningInNetlify ? '@astrojs/netlify/image-service.js' : undefined,
+							entrypoint: enableImageCDN ? '@astrojs/netlify/image-service.js' : undefined,
```

Comment:
Is it safe to set the service to `undefined`? Shouldn't we set the service only if `enableImageCDN` is `true`?

Reply:
IIFRC, `undefined` equals the default one